backend/src/main/java/com/quiz/converter/services/MoodleXmlCreatorService.java

```diff
@@ -65,10 +64,17 @@ private static void addQuestion(Question question, Document doc, Element rootEle
     }
 
     private static void addQuestionOptions(Question question, Document doc, Element questionElem) {
-        var shuffleTagElem = doc.createElement("shuffleanswers");
-        var shuffleText = question.shuffle() ? "1" : "0";
-        shuffleTagElem.setTextContent(shuffleText);
-        questionElem.appendChild(shuffleTagElem);
+        if (question.type().equals(QuestionType.REGULAR_EXPRESSION)) {
```

Comment:
Also TEXT_MATCH?

Reply:
.

Reply:
Text match doesn't need "studentshowalternate" element for it to work.